tools/mac_setup.sh

```diff
@@ -55,15 +53,30 @@ export PATH="$PATH:/usr/local/bin"
 
 # openpilot environment
 if [ -z "$OPENPILOT_ENV" ] && [ -n "$RC_FILE" ] && [ -z "$CI" ]; then
-  echo "export PATH=\"\$PATH:$HOME/.cargo/bin\"" >> $RC_FILE
   echo "source $ROOT/tools/openpilot_env.sh" >> $RC_FILE
-  export PATH="$PATH:\"\$HOME/.cargo/bin\""
   source "$ROOT/tools/openpilot_env.sh"
   echo "Added openpilot_env to RC file: $RC_FILE"
 fi
 
 # install python dependencies
-$ROOT/update_requirements.sh
+$ROOT/update_requirements.sh || true
+
+# install casadi
+echo "-- casadi manual install"
```

Comment:
do we need to do this for the x86 macs?

Reply:
actually, I'm fine with keeping as is so it has CI coverage

Reply:
i dont think so, but the pipfile doesnt have a way to distinguish b/t architecture except for the host OS (markers in the pipfile) afaik